src/Classes/Factories/DisplayApplicantsControllerFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Portal\Factories;
+
+use Psr\Container\ContainerInterface;
+use Portal\Controllers\DisplayApplicantsController;
+
+class DisplayApplicantsControllerFactory
+{
+    /**
+     * Instantiates DisplayApplicantsController with dependencies.
+     *
+     * @param ContainerInterface $container
+     *
+     * @return DisplayApplicantsController
+     */
+    public function __invoke(ContainerInterface $container) : DisplayApplicantsController
+    {
+        $renderer = $container->get('renderer');
+        $applicantModelFactory = $container->get('ApplicantModel');
```

Comment:
change this to applicant model - otherwise it is a bit confusing - you have called on the factory to create the applicant model and so we are inserting the model not the factory